spec/controllers/api/v1/urls_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::UrlsController, type: :controller do
+
+  it "can shorten a url" do
+    post :create, format: :json, url: { long: 'http://jumpstartlab.com' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.